mne_bids/write.py

```diff
@@ -1514,12 +1546,13 @@ def mark_bad_channels(ch_names, descriptions=None, *, bids_path,
     # Update info['bads']
     bads = _get_bads_from_tsv_data(tsv_data)
     raw.info['bads'] = bads
-    # XXX (How) will this handle split files?
     if isinstance(raw, mne.io.brainvision.brainvision.RawBrainVision):
-        events, event_id = _read_events(events_data=None, event_id=None,
-                                        raw=raw, verbose=False)
+        # XXX We should write durations too, this is supported by pybv.
```

Comment:
fixed now?

Reply:
No, we should address this in a followup PR, as it would require changes in `_write_raw_branvision` too. I'd like to keep it out of this PR.

The `_events.tsv` sidecars are written correctly in any case.